CMakeLists.txt

```diff
@@ -337,7 +337,7 @@ if(TESTBUILD_OPENCL_PROGRAMS)
       message(STATUS "Found clang compiler - ${CLANG_OPENCL_COMPILER}")
 
       find_path(CLANG_OPENCL_INCLUDE_DIR opencl-c.h
-        HINTS ${LLVM_INSTALL_PREFIX}/lib/clang ${LLVM_INSTALL_PREFIX}/lib64/clang
+        HINTS ${LLVM_INSTALL_PREFIX}/lib/clang ${LLVM_INSTALL_PREFIX}/lib64/clang /usr/lib/clang /usr/lib64/clang
```

Comment:
Any reason the variable is not available on Gentoo? Maybe better to address upstream?

Reply:
To be honest, I'm not really sure about this one too. I can also drop it for now and see if there are any issues without it in Gentoo.

Reply:
https://github.com/darktable-org/darktable/pull/1599 / https://github.com/gentoo/gentoo/pull/6633
I guess we can drop this one...

Reply:
dropped